.sync.yml

```diff
@@ -0,0 +1,49 @@
+---
+.travis.yml:
```

Comment:
Abstracting the travis config through PDK seems to make things more complicated and harder to debug. Unless you see a big win here, I'd rather set this to unmanaged.

Reply:
It's simpler to do updates of like rvm and adding new OSes to support, just update .sync.yml for new OS and update one place for new RVM.  If we continue to use PDK I'd recommend forking puppetlabs then merging changes from my fork that I used: https://github.com/treydock/pdk-templates.  I am not as keen on keeping RVM versions up-to-date so thought if that's of value might be worth adding a fork to this namespace where this and other modules could pull PDK changes from

Reply:
Changes to rvm versions only happen in travis config though right, not in a bunch of places, so I don't see why adding them to sync is advantageous over travis. Perhaps I'm missing something?

Reply:
In the pdk-template you update the RVM version in one place, then run `pdk update` and this module and every other module using PDK template will get updated. Just keeps things centralized.